src/Types/TypedTxOut.purs

```diff
@@ -0,0 +1,289 @@
+module Types.TypedTxOut
```

Comment:
This whole module is to make the typed validators possible and there is no analog in Plutus since it's not necessary, right? If so, can we add a doc comment to the module explaining this?

Reply:
https://playground.plutus.iohkdev.io/doc/haddock/plutus-ledger/html/src/Ledger.Typed.Tx.html This is the equivalent module. I can add more comments.